src/components/tasks/card/index.tsx

```diff
@@ -1,10 +1,15 @@
-import { FC, useState } from 'react';
-import Image from 'next/image';
-import classNames from '@/components/tasks/card/card.module.scss';
-import task from '@/interfaces/task.type';
-import { AVAILABLE, BLOCKED, COMPLETED, VERIFIED } from '@/components/constants/beautified-task-status';
+import { FC, useState } from "react";
+import Image from "next/image";
+import classNames from "@/components/tasks/card/card.module.scss";
+import task from "@/interfaces/task.type";
+import {
+  AVAILABLE,
+  BLOCKED,
+  COMPLETED,
+  VERIFIED,
+} from "@/components/constants/beautified-task-status";
 
-const moment = require('moment');
+const moment = require("moment");
```

Comment:
Can we please use single quotes as they were before throughout the changed files ? You can change the settings of your local prettier in the VSCode settings

Reply:
Done!